www/chromedriver/Portfile

```diff
@@ -32,129 +56,159 @@ use_configure       no
 build {}
 
 destroot {
-    xinstall -m 755 \
-        ${worksrcpath}/chromedriver \
+    copy \
```

Comment:
Any reason to not use `xinstall`?

Reply:
Doesn’t work for symlinks.

Reply:
maybe when `file move`? It is much faster and should be much safer for symlinks.

Reply:
Not sure what you mean. This just copies a symlink to `destroot`, which is what we need here.

Reply:
Also fwiw it copies the actual file without the `undetected` variant. This code works for both.